app/register/mars-rover-sprite.svg

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
If both mars rover sprites are the same, I'd have it exist in the root directory of the website rather than duplicating it